core/js/sharedialogview.js

```diff
@@ -95,6 +95,9 @@
 		/** @type {object} **/
 		shareeListView: undefined,
 
+		/** @type {string} **/
+		batchActionSeparator: ', ',
```

Comment:
```suggestion
		batchActionSeparator: ',',
```

This could be just a comma, and the code could normalize the input by removing any white-space (thus allowing users to put white-space or not)